### PR TITLE
Cooldown/upgrade

### PR DESCRIPTION
**Pull request by pascalbetz**

### CODE REVIEW

**Review by ZmagoD — APPROVED** [review]
not much to review here 😄 
I will take a deeper loot at the review app

File: db/migrate/20200423073425_create_alchemy_essence_nodes.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 class CreateAlchemyEssenceNodes < ActiveRecord::Migration[5.2]
   def change
-    create_table :alchemy_essence_nodes do |t|
-      t.references "node"
-      t.timestamps
-    end
-    add_foreign_key "alchemy_essence_nodes", "alchemy_nodes", column: "node_id"
+    # create_table :alchemy_essence_nodes do |t|
```
[the remark this answers is not in the document]
Comment (pascalbetz) [reply db/migrate/20200423073425_create_alchemy_essence_nodes.rb:5]: because it seems that this was already executed on our installation....

**Comment by dancristianb** [comment]
@Hassanelashram @pascalbetz @ZmagoD   the `alchemy_cms` Gemfile config is now pointing towards the `cooldown/upgrade` branch https://github.com/algrano/webapp/blob/master/Gemfile#L168

Is there a preference on merging this in and have the app point towards [rails-6.1](https://github.com/algrano/alchemy_cms/tree/rails-6.1) again or are we OK with closing this?

**Comment by ZmagoD** [comment]
@dancristianb for now it is OK. We plan to get rid of this fork anyway

**Comment by dancristianb** [comment]
Closing based on https://github.com/algrano/alchemy_cms/pull/4#issuecomment-1092778191 and https://github.com/algrano/alchemy_cms/pull/4#issuecomment-1092913944

**Pull request closed without merging**